source/transactions/tests/unified/mongos-unpin.yml

```diff
@@ -116,3 +116,17 @@ tests:
       - *assertPinnedServer
       - *insertOne
       - *assertNoPinnedServer
+
+  - description: unpin when a non-transaction read operation uses a session
+    operations:
+      - *startTransaction
+      - *insertOne
+      - name: commitTransaction
+        object: *session0
+      - *assertPinnedServer
+      - name: find
+        object: *collection0
+        arguments:
+          filter: { x: 1 }
+          session: *session0
+      - *assertNoPinnedServer
```

Comment:
nit: The file should have a blank line at the end.

Reply:
I don't know what's going on here.  I saw this, added a blank line, squashed and force pushed, and still seeing this.

Reply:
Just to see if it'd work, I tried modifying this PR by adding a blank line and it seems to have fixed the issue. Let me know if you'd like me to revert the commit and put the PR back where it was, though.